sql/plan.go

```diff
@@ -136,6 +136,9 @@ type planNode interface {
 	Next() bool
 	// Err returns the error, if any, encountered during iteration.
 	Err() error
+	// ExplainPlan fills in name and description for the node and returns a list
+	// of child nodes.
+	ExplainPlan(name, description *string) []planNode
```

Comment:
I feel like out-parameters is a pretty exotic choice in Go, so I'd rather avoid it. It doesn't buy clarity since the caller still has to assign names to positional arguments, so I'd rather we used multiple return here.